config/rootfs/chromiumos/scripts/build_board.sh

```diff
@@ -32,6 +32,17 @@ echo Board ${BOARD} setup
 # Compiling ChromiumOS image
 # Future possible option --profile=x, for example kernel-5_15, profiles are at /mnt/host/source/src/overlays/overlay-${BOARD}/profiles/
 cros_sdk setup_board --board=${BOARD}
+
+# Without workarounds hatch and octopus build failing
```

Comment:
The commit message doesn't match the changes in this commit.

Reply:
Probably should be fixed now